presto-docs/src/main/sphinx/connector/elasticsearch.rst

```diff
@@ -185,6 +185,51 @@ Elasticsearch Presto
 (all others)  (unsupported)
 ============= =============
 
+Array Types
+^^^^^^^^^^^
+
+Elasticsearch has `no dedicated array type <https://www.elastic.co/guide/en/elasticsearch/reference/current/array.html>`_ so the `_meta field <https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-meta-field.html>`_ of the index is used to store an property structure that contains
```

Comment:
This sentence is a bit long. I'd restructure as follows (or something along those lines): "Fields in Elasticsearch can contain zero or more values, but there is no dedicated array type. To indicate a field contains an array, it can be annotated in a presto-specific structure in the _meta section of the index mapping."